includes/utilities.inc

```diff
@@ -460,11 +460,13 @@ function islandora_paged_content_ocr_combine(array $files, $out) {
   $success = TRUE;
   foreach ($files as $file) {
     $ocr = file_get_contents($file);
-    if ($ocr == FALSE) {
+    if ($ocr === FALSE) {
       $success = FALSE;
     }
     else {
-      $success = file_put_contents($out, $ocr, FILE_APPEND) == FALSE ? FALSE : $success;
+      if ($ocr) {
```

Comment:
This bears somewhat the same problem, no? For example, if `$ocr` simply consisted of the number 0, `$success` would remain true despite the OCR not being appended.

I think this `if ($ocr) {}` logic can be dropped in favour of simply making a strict comparison between `FALSE` and the results of `file_put_contents()` on the next line instead so that false-y strings can be appended

Reply:
bump ... @qadan - sorry this is such an old issue now, but I am confused by the merge from @whikloj  https://github.com/Islandora/islandora_paged_content/pull/133 ... is that merged?  Does this still need anything else merged or merged at all?

Reply:
@bgilling I think you just need to try adding @qadan's code and test the results... If all is well and @qadan approves, I can take a final stab at testing and merge it.